packages/scxa-marker-gene-heatmap/src/heatmapOptionsProvider.js

```diff
@@ -1,4 +1,26 @@
 const heatmapOptionsProvider = {
+  cellmultiexperiment: {
+    tooltip: {
+      // followPointer: true,
+      formatter: function () {
+        if(this.point.value === null) {
+          return `<b>Gene ID:</b> ${this.point.geneName}<br/>` +
+              `<b>Expression:</b> Not expressed<br/>`
+        }
+        else {
+          const text =
+              `<b>Cell marker gene from experiment:</b> ${this.point.cellGroupValueWhereMarker}<br/>` +
+              `<b>Gene ID:</b> ${this.point.geneName}<br/>` +
+              `<b>Expression:</b> ${+this.point.value.toFixed(3)} CPM`
+          return text
+        }
+      }
+    },
+    title: cellType => `${cellType} marker genes`,
+    labelsFormatter: label => label,
+    noData: `No marker genes found for the selected organ or region. Try selecting another organism part.`
```

Comment:
Rephrase to: “No marker enes found for the selected cell type. Try selecting a different cell type.”.